API/src/utils/Config/index.js

```diff
@@ -18,9 +18,9 @@ module.exports = {
   database: {
     host: process.env.DATABASE_HOST || `localhost`,
     port: process.env.DATABASE_PORT || 5432,
-    username: process.env.DATABASE_USERNAME || `postgres`,
-    password: process.env.DATABASE_PASSWORD || `paul`,
-    name: process.env.DATABASE_NAME || `risk`,
+    username: process.env.DATABASE_USERNAME || `cat`,
+    password: process.env.DATABASE_PASSWORD || `1234`,
+    name: process.env.DATABASE_NAME || `OCAT`,
```

Comment:
```suggestion
    username: process.env.DATABASE_USERNAME || config.database.username,
    password: process.env.DATABASE_PASSWORD || config.database.password,
    name: process.env.DATABASE_NAME || config.database.name,
```
This is my fault.  When I was making the project easier, I rushed and didn't do this the right way!  Proof that even experienced folks can make mistakes :)

This is a somewhat large security concern.  If you're supplying passwords / usernames, you shouldn't do it in your code.  On a production application, this could be lost in a data breach.  Hackers could access your production database...etc.  To get around this, we pull this data from the config.json and then don't add that config.json to git (place it in the .gitignore)

Reply:
@BenjaminGomori Did you see this comment?

Reply:
Yes, sorry.
I changed this as recommended.